src/components/camera/feed-watermark.tsx

```diff
@@ -0,0 +1,31 @@
+import { useAtom } from "jotai";
+import { userAtom } from "@/state/user-atom";
+
+export const CameraFeedWatermark = () => {
+  const [user] = useAtom(userAtom);
```

Comment:
https://jotai.org/docs/core/use-atom#useatomvalue

```suggestion
  const authUser = useAtomValue(authUserAtom);
```